recipe/meta.yaml

```diff
@@ -7,7 +8,7 @@ package:
 source:
   fn: rasterio-{{ version }}.tar.gz
   url: https://github.com/mapbox/rasterio/archive/{{ version }}.tar.gz
-  sha256: 472908e335fc9a4d072fb6f41e08b58eb7120f18f6c12e28ffd3beb86edd7e1f
+  sha256: {{ sha256 }}
```

Comment:
@ceholden I am not a fan of jinjafying everything. The version number is handy but the `checksum` is something we change in only one place anyways. Also, if we decided to move away from `sha256`, like when we moved from `md5`, this variable makes it awkward. In addition to that I have some script that helps me automate the updates and the script expects the checksum to be here to avoid parsing the recipe twice.

With all that said, if you really want this change I can live with it, but I thought I should let you know why I chose the other format.